web/src/containers/Admin/Broker/index.js

```diff
@@ -6,7 +6,8 @@ import BrokerForm from './BrokerForm';
 import { getConstants, updatePlugins } from '../Plugins/action';
 import { getFees } from '../Fees/actions';
 import { validateRequired } from '../../../components/AdminForm/validations';
-import { formatCurrency } from '../../../utils/currency';
+import { formatCurrencyByIncrementalUnit } from '../../../utils/currency';
+import { connect } from 'react-redux';
```

Comment:
Just as a convention, it's better to import external libraries at the top of local components and functions.